CMakeLists.txt

```diff
@@ -85,6 +87,19 @@ add_compile_options(/WX /Gy
     # note that /Zi generates debug info inside the object file, it's the same as /Z7 for msvc
     "$<$<COMPILE_LANGUAGE:ASM_MASM>:/Zi;/W3;/nologo>")
 
+if(STL_USE_ANALYZE)
+    # warning C6553: The annotation for function 'LCMapStringEx' on _Param_(9)
+    # does not apply to a value type.
+    # There's a bug in the declaration for LCMapStringEx - it applies _In_opt_ to an LPARAM.
+    # LPARAM is a LONG_PTR (intptr_t), and it's invalid to apply _In_opt_ to a non-pointer.
+

```

Comment:
Removing the C6553 suppression and passing `-k 1000` to Ninja reveals one occurrence in the STL itself, where we copied a WinSDK bug, `_In_opt_ DWORD const msWindowLength`: https://github.com/microsoft/STL/blob/7f04137880e1c3df5125c1baf808f16f399dee2e/stl/src/winapisupp.cpp#L195-L196
This emits `warning C6553: The annotation for function '__crtSetThreadpoolTimer' on _Param_(4) does not apply to a value type.`